.gitignore

```diff
@@ -8,12 +9,17 @@
 /spec/reports/
 spec/internal
 /tmp/
+/config
+
+#Ignore config and intermediate file extensions
 *.bundle
 *.so
 *.o
 *.a
+*.swp
```

Comment:
Put this in your global git ignore. See https://help.github.com/articles/ignoring-files/#create-a-global-gitignore